lib/src/authentication/src/client_options.dart

```diff
@@ -195,6 +187,83 @@ class ClientOptions extends AuthOptions {
   /// https://docs.ably.com/client-lib-development-guide/features/#TO3l7
   int channelRetryTimeout = 15000;
 
+  /// Initializes an instance with defaults
+  ClientOptions({
```

Comment:
If the order of these parameters is arbitrary (that is, given they are named, their order carries no meaning) then I wonder if we should consider sorting them alphabetically. 🤔 

On a quick search I can't see any evidence there are any lint rules that can enforce this for us, so perhaps not worth it. Just an idea.

Reply:
I don't think there's a lint rule for that, as far as I know Dart doesn't define this, but I think it may be a nice idea to have them sorted in all class constructors. However, I'd rather not do it in scope of this PR 🙂

Reply:
Fair enough and a good call. Perhaps create an issue for this, @ikurek, as a potential code quality improvement for the backlog?

Reply:
It's created: https://github.com/ably/ably-flutter/issues/351 🙂